sway-ir/tests/dce/dce2.cfg

```diff
@@ -0,0 +1,16 @@
+// regex: ID=[[:alpha:]0-9]+
+
+script {
+    fn main() -> bool {
+        entry:
+        v0 = const u64 11
+        v1 = const u64 0
+        br block0
+
+        block0:
+// not: phi(entry: $(ID))
```

Comment:
I don't think we should be DCEing `phi`s.  If we need to add a value to the `phi` later due to `simplify-cfg` or something then it'll get confused, as it's generally assumed they exist.

Removing them probably isn't really going to be beneficial in the grander scheme of things anyway..?  We should special case them, and perhaps aim to convert them from being an instruction into instead being an intrinsic part of a `Block` in the future.

Reply:
I'm not removing the PHIs. I'm replacing them with one without any input values.

Reply:
Ahh, OK, I missed that. :+1:

Reply:
The IR printer doesn't print empty PHIs (this had be quite puzzled a few days back when I first learnt about it. I was hitting an error telling me that there's a PHI, and the IR I printed for debugging didn't show it :man_shrugging:).

Reply:
Yeah, it's a little bit of preemptive DCE, pretending they're not there. :slightly_smiling_face: